<launch>

  <arg name="model" default="burger"/>

  <node name="map_building1" pkg="self_topoexplore" type="robot_rela_pose.py" output="screen">
    <param name="network" value="gl18-tl-resnet101-gem-w" />
    <param name="platform" value="cuda" />
    <param name="robot_name" value="robot1"/>
    <param name="robot_num" value="2"/>
  </node>

  <node name="map_building2" pkg="self_topoexplore" type="robot_rela_pose.py" output="screen">
    <param name="network" value="gl18-tl-resnet101-gem-w" />
    <param name="platform" value="cuda" />
    <param name="robot_name" value="robot2"/>
    <param name="robot_num" value="2"/>
  </node>

  <node name="experiment" pkg="self_topoexplore" type="experiment.py" output="screen">
    <param name="robot_num" value="2"/>
  </node>

</launch>